python/tests/test_sync.py

```diff
@@ -0,0 +1,49 @@
+import unittest
```

Comment:
Tests for this repository should be written using the `pytest` framework, you can check some of the sibling files for examples. Given the content of this file, it would probably be better to add these tests for `reorder_modules` into `test_parsing` -- also because `reorder_modules` should be moved into the `parsing` module instead of `sync`.